activerecord/lib/active_record/database_configurations/hash_config.rb

```diff
@@ -87,6 +87,13 @@ def idle_timeout
       def adapter
         configuration_hash[:adapter]
       end
+
+      # The path to the schema cache dump file for a database.
+      # If omitted, the filename will be read from ENV or a
+      # default will be derived.
+      def schema_cache_path
+        configuration_hash[:schema_cache_path]
+      end
```

Comment:
Oh I just realized there's one part missing. Can you add a not implemented version of this method in https://github.com/rails/rails/blob/ab3c34e978ca5843cf936d1ea298d543a34df584/activerecord/lib/active_record/database_configurations/database_config.rb?